autoscheduler/frontend/src/tests/ui/SchedulePreview.test.tsx

```diff
@@ -281,10 +282,70 @@ describe('SchedulePreview component', () => {
       fireEvent.click(confirmDeleteButton);
 
       // assert
-      const { allSchedules, savedSchedules } = store.getState().schedules;
-      expect(savedSchedules).toHaveLength(0);
-      expect(allSchedules).toHaveLength(1);
-      expect(allSchedules[0]).toEqual(testSchedule1);
+      const { schedules } = store.getState();
+      expect(schedules).toHaveLength(1);
+      expect(schedules[0].saved).toBe(false);
+      expect(schedules[0].meetings).toEqual(testSchedule1);
+    });
+  });
+
+  describe('renames the correct schedule', () => {
```

Comment:
I have a few tests that schedules are renamed correctly in the redux tests (not specifically for `getUniqueNames`, but for the `renameSchedule` action). For redux, I consider a unit test to be a test that dispatching an event gives correct output (black box testing each action) rather than if a particular helper function doing its job . Since we definitely don't want to remove tests that the action works correctly, I think it's a bit redundant to have tests for the helper as well? That's my reasoning at least, let me know if you feel differently